.github/workflows/release.yaml

```diff
@@ -0,0 +1,275 @@
+name: Release
```

Comment:
I didn't see this before adding `RELEASING.md`, sorry. Will move to `README.md` here

I added git-cliff just to match what flopy does. I could remove it. It works best if commits are [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary), but can be [configured](https://git-cliff.org/docs/configuration#filter_unconventional) to accept any format with `conventional_commits = false`. With the defaults

```
conventional_commits = true
filter_unconventional = true
```

I don't think you'd need to squash since merge commits would be filtered out.

Reply:
I think the README.md is good. It being in `ci/` seems ok since this isn't something needed by anyone but us since we are the only ones that will make releases.